.github/workflows/autodocs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     
     - name: Install Launcher
       id: install-launcher
-      run: cargo install --git git+https://github.com/${{ github.repository }}/text-generation-inference.git --branch ${{ github.head_ref }} text-generation-launcher
+      run: cargo install --git git+https://github.com/${{ github.repository }} --branch ${{ github.head_ref }} text-generation-launcher
```

Comment:
I spent two hours trying to get this right. 
![image](https://github.com/huggingface/text-generation-inference/assets/53175384/95aff675-8085-4ee7-bfd0-3e2c76508f9d)